test/create-user-test.ts

```diff
@@ -0,0 +1,144 @@
+import { describe, it, afterEach } from 'mocha';
+import { expect } from 'chai';
+import axios from 'axios';
+import { prisma } from '../src/database.js';
+import { UserInput } from '../src/resolvers/create-user-resolver.js';
+import * as bcrypt from 'bcrypt';
+import {
+  EXISTING_EMAIL_MESSAGE,
+  INVALID_BIRTH_DATE_MESSAGE,
+  INVALID_PASSWORD_MESSAGE,
+} from '../src/utils/validators.js';
+import { LOCAL_SERVER_URL } from './index.js';
+
+const createUserMutationRequest = (email: string, name: string, password: string, birthDate: string) => {
```

Comment:
vc já ta usando o `UserInput` na linha 24 pra tipar, poderia usar já aqui para tipar o parametro como um objeto, não?
```suggestion
const createUserMutationRequest = (input: UserInput) => {
```

Reply:
Tinha ficado na dúvida como era melhor quando fiz kkkk, já alterei aqui